Python/Embeddings/RegExp (for Python).sublime-syntax

```diff
@@ -25,21 +29,30 @@ contexts:
 
   group-start:
     - meta_prepend: true
-    - match: (\?P=)([a-zA-Z_][a-zA-Z_0-9]*\w*)(?=\))
+    - match: (\?P=)({{capture_group_name}})(?=\))
       captures:
         1: keyword.other.back-reference.named.regexp
         2: variable.other.backref-and-recursion.regexp
-    - match: (\?P)(<)([a-z]\w*)(>)
+    - match: (\?P)(<)({{capture_group_name}})(>)
       captures:
         1: keyword.other.backref-and-recursion.regexp
         2: punctuation.definition.capture-group-name.begin.regexp
         3: entity.name.capture-group.regexp
         4: punctuation.definition.capture-group-name.end.regexp
-    - match: (\?)(\()([1-9][0-9]?|[a-zA-Z_][a-zA-Z_0-9]*)(\))
-      # We can make this more sophisticated to match the | character that separates
-      # yes-pattern from no-pattern, but it's not really necessary.
+    # We can make this more sophisticated to match the | character that separates
+    # yes-pattern from no-pattern, but it's not really necessary.
+    - match: (\?)(\()({{capture_group}})(\))
       captures:
         1: keyword.other.backref-and-recursion.conditional.regexp
         2: punctuation.definition.group.begin.assertion.conditional.regexp
         3: variable.other.back-reference.regexp
         4: punctuation.definition.group.end.assertion.conditional.regexp
+
+  backrefs:
+    - meta_prepend: true
+    - match: \\g(<)({{capture_group}})(>)
+      scope: keyword.other.backref-and-recursion.regexp
```

Comment:
The other patterns only seem to scope initial part (e.g. `?P=` or `?P`) keyword, while this pattern scopes `\g` as well as whole name part `keyword`.

Maybe we should decide for one style.

Scoping everything would make it easier to address the whole expression with single simple selector, while it on the other hand seems a bit weird to scope actual name `keyword variable`.

Reply:
OK, normal Regexp.sublime-syntax also scopes it this way. Maybe the other ones `?P=` need some adjustmant to align them. Can be modified in a separate PR however.

Reply:
I agree with you. I don't like the status quo, but I think we should RFC that.